src/openrct2/actions/RideSetStatus.hpp

```diff
@@ -88,7 +88,9 @@ DEFINE_GAME_ACTION(RideSetStatusAction, GAME_COMMAND_SET_RIDE_STATUS, GameAction
         }
 
         res->ErrorTitle = _StatusErrorTitles[_status];
-        ride->FormatNameTo(res->ErrorMessageArgs.data() + 6);
+
+        Formatter ft(res->ErrorMessageArgs.data() + 6);
```

Comment:
Not sure how i missed this but should be `ft(res->ErrorMessageArgs.data()); ft.Increment(6);` I plan to change ErrorMessageArgs into a formatter eventually and that will make it easy to change.

Same goes for the one below in the execute.